.NET/Samples/SimpleConsole_Core/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.DateTime;
+using Microsoft.Recognizers.Text.Number;
+using Microsoft.Recognizers.Text.NumberWithUnit;
+using Microsoft.Recognizers.Text.Sequence;
+using Newtonsoft.Json;
+
+namespace SimpleConsole_Core
```

Comment:
Do we really need two SimpleConsole? I think we can keep just one, and that one can be a .NET core project, as we want to make sure the .NET standard version of the recognizers is the preferred target.

Reply:
Please make another small PR with just this change, as it's unrelated to the new recognizer.

Reply:
Sample removed